awx/main/scheduler/task_manager.py

```diff
@@ -87,7 +87,21 @@ def after_lock_init(self):
         instances_by_hostname = {i.hostname: i for i in instances_partial}
 
         for rampart_group in InstanceGroup.objects.prefetch_related('instances'):
-            self.graph[rampart_group.name] = dict(graph=DependencyGraph(), capacity_total=rampart_group.execution_capacity, consumed_capacity=0, instances=[])
+            self.graph[rampart_group.name] = dict(
+                graph=DependencyGraph(),
+                execution_capacity=0,
+                control_capacity=0,
+                consumed_capacity=0,
```

Comment:
We don't count total available capacity, but we do count total consumed capacity. This is only used for writing log messages, which I don't want to muck with.

In the API, we do report total capacity, which is also something I don't want to mess with, because it should be the same as execution capacity in all cases except for the control plane group.